src/templates/layouts/vetCenter/index.tsx

```diff
@@ -0,0 +1,344 @@
+
+                    {/* TODO: potential change to the vets-website widget to not use facilityId as a prop on div */}
+                    {/* <div
+                      data-widget-type={`expandable-operating-status-${fieldFacilityLocatorApiId}`}
+                      facilityId={fieldFacilityLocatorApiId}
+                      status={operatingStatusFacility}
+                      info={operatingStatusMoreInfo}
+                    /> */}
+
+                    <div className="vads-u-margin-bottom--3">
+                      <address>
+                        <div>{address.address_line1}</div>
+                        {address.address_line2 && (
+                          <div>{address.address_line2}</div>
+                        )}
+                        <div>{`${address.locality}, ${address.administrative_area} ${address.postal_code}`}</div>
+                      </address>
+                      <GoogleMapsDirections
+                        title={title}
+                        address={directionsString}
+                      />
+                    </div>
+
+                    <h4 className="vads-u-font-size--lg vads-u-margin-top--0 vads-u-line-height--1 vads-u-margin-bottom--1">
+                      Direct line
+                    </h4>
+                    <div className="vads-u-margin-bottom--3">
+                      <div className="main-phone vads-u-margin-bottom--1">
+                        <a href={`tel:${phoneNumber}`}>{phoneNumber}</a>
+                      </div>
+                    </div>
+
+                    <Hours headerType="standard" allHours={officeHours} />
+                  </section>
+                </div>
+              </div>
+            </div>
+            <ImageAndStaticMap
+              image={image}
+              facilityId={fieldFacilityLocatorApiId}
+            />
+          </div>
+
+          {/* Other locations */}
+          <div className="vads-u-margin-bottom--3">
+            <h3 className="vads-u-font-size--lg vads-u-line-height--1 vads-u-margin-bottom--1">
+              Other Locations
+            </h3>
+            <div>
+              <p className="vads-u-margin-bottom--0 vads-u-line-height--4">
+                Vet Centers are community based to be more accessible in areas
+                where you live.
+              </p>
+              <a href={`${path}/locations`}>View more {title} locations</a>
+            </div>
+          </div>
+
+          {ccNonTraditionalHours && (
+            <div
+              className="vads-u-font-weight--bold"
+              id="field-cc-non-traditional-hours"
+            >
+              <WysiwygField html={ccNonTraditionalHours.html} />
+            </div>
+          )}
+
+          {/* Call Center Information */}
+          {ccVetCenterCallCenter && (
+            <div className="vads-u-margin-bottom--2">
+              <AlertBlock
+                alertType="info"
```

Comment:
I called out the alert color, but it looks like the gray alert is removed in V3, so nevermind: https://design.va.gov/components/alert